docs/scos/dev/tutorials-and-howtos/introduction-tutorials/tutorial-content-and-search-attribute-cart-based-catalog-personalization-spryker-commerce-os/tutorial-content-and-search-attribute-cart-based-catalog-personalization-spryker-commerce-os/tutorial-boosting-cart-based-search-tutorial-content-and-search-attribute-cart-based-catalog-personalization.md

```diff
@@ -0,0 +1,384 @@
+---
```

Comment:
Why is this file part of the PR for the Configurable Product in the Wish List?

Reply:
On my machine. Jekyll does not run correctly when files have too long titles (windows problem), so I have to shorten the title each time and then rename it back.